components/docs-chef-io/content/automate/node_credentials.md

```diff
@@ -44,6 +44,27 @@ Select _Add Credential_ and a dialog box appears as shown below. Select the _Cre
 
 **WinRM** requires a credential name, a user name, and a WinRM password.
 
+Windows machines **must have** the following configurations:
+
+* Ports 3389(RDP), 80(HTTP), 443(HTTPS), 5985(WinRM) and 5986(WinRM) must be open.
```

Comment:
must be open and reachable from Chef Automate

Reply:
@vivek-yadav, made the requested change